readme.txt

```diff
@@ -30,8 +30,54 @@ Extract the contents of the ZIP and upload the contents to the `/wp-content/plug
 
 Data is coming from [The Green Web Foundation](https://www.thegreenwebfoundation.org/) API.
 
+== Security ==
+
+For security or privacy concerns, please contact us at [wordpress @ wpsysadmin . com].
+
+This plugin adheres to the following security measures and review protocols for each version:
+
+* [WordPress Plugin Handbook](https://developer.wordpress.org/plugins/)
+* [WordPress Plugin Security](https://developer.wordpress.org/plugins/wordpress-org/plugin-security/)
+* [WordPress APIs Security](https://developer.wordpress.org/apis/security/)
+* [WordPress Coding Standards](https://github.com/WordPress/WordPress-Coding-Standards)
+
+== Privacy ==
+
+* This plugin does not collect any information about your site, your identity, the plugins, themes or content the site has.
+
+== Vulnerabilities ==
+
+* No vulnerabilities have been published up to version 2.2.1.
+
```

Comment:
Perhaps we should include a way for people to report security vulnerabilities.

How about this?
```suggestion
Found a security vulnerability? Please report it to us privately at the [WPsustainable GitHub repository](https://github.com/javiercasares/WPsustainable/security/advisories/new).
```

Reply:
I like this idea...